sdformat_mjcf/src/sdformat_mjcf/mjcf_to_sdformat/converters/geometry.py

```diff
@@ -43,7 +43,12 @@ def mjcf_geom_to_sdf(geom):
     sdf_geometry = sdf.Geometry()
     if geom.type == "box":
         box = sdf.Box()
-        box.set_size(su.list_to_vec3d(geom.size) * 2)
+        if geom.fromto is None:
+            box.set_size(su.list_to_vec3d(geom.size) * 2)
+        else:
+            v1 = Vector3d(geom.fromto[0], geom.fromto[1], geom.fromto[2])
+            v2 = Vector3d(geom.fromto[3], geom.fromto[4], geom.fromto[5])
+            box.set_size((v2 - v1).abs())
```

Comment:
I tested the behavior of using `fromto` for boxes in Mujoco and it's different 
from this. From what I can tell, it uses the fromto for the position and 
orientation of the box as one would expect, but it also uses the vector's length as the length of one dimension of the box and the `size` attribute as the lengths in the other dimensions.

Since this is not a well documented specification, I'd be okay with making this 
an error.